Tasks/AzureFunctionAppV2/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -32,7 +32,8 @@ const linuxFunctionRuntimeSettingValue = new Map([
     [ 'PYTHON|3.6', 'python '],
     [ 'PYTHON|3.7', 'python '],
     [ 'PYTHON|3.8', 'python '],
-    [ 'PYTHON|3.9', 'python ']
+    [ 'PYTHON|3.9', 'python '],
+    [ 'PYTHON|3.10', 'python ']
```

Comment:
>3.10', 'pytho [](http://example.com/codeflow?start=14&length=13)

Is this change relevent?

Reply:
It's not specifically relevant for the bearer token change but is a change that we need for the v2 task as Python 3.10 is fully supported.